recipes-app/node-red-gpio/files/0001-Dynamic-add-user-button-label.patch

```diff
@@ -0,0 +1,35 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Thu, 4 May 2023 10:31:09 +0800
+Subject: [PATCH] Dynamic add user button label
+
+Not all platform has the user button supported, so dynamic
+to add the user button label.
```

Comment:
Sorry, does not parse yet for me: How exactly is that determined if the button is there or not? And what is "dynamic" about this?

Reply:
Adaptative for different board type.
This is fix for:
https://github.com/node-red/node-red-nodes/blob/25d7d668ef1123d1182ece9e6c2ccd4ddc3d3c03/hardware/intel/mraa-gpio-din.html#L71.

We add the user button for the IOT2050, but the raspi or organe pi don't have the user button label before,
This should be the specific for the iot2050.   So its need to according to the iot2050 board type to dynamic add this user button node.

Reply:
So, we broke this in upstream? Then please enhance the commit message accordingly so that we also submit it.